korge/src/commonMain/kotlin/com/soywiz/korge/view/animation/ParallaxDataView.kt

```diff
@@ -275,7 +275,8 @@ class ParallaxDataView(
     }
 }
 
-suspend fun VfsFile.readParallaxDataContainerFromAseFile(
+suspend fun VfsFile.readParallaxDataContainer(
+    formats: ImageFormat = RegisteredImageFormats,
     config: ParallaxConfig,
```

Comment:
Since it is optional, maybe would be nice to move it after config. And since `ImageFormat` is singular, maybe the parameter could be renamed to `format`

Reply:
You are right. I even can set it to "ASE" instead to "RegisteredImageFormats".